File: prop-all/src/test/java/io/pleo/prop/objects/EmptyNamedAnnotation.java
package io.pleo.prop.objects;

import javax.inject.Inject;
import javax.inject.Named;

import io.pleo.prop.core.Prop;

public class EmptyNamedAnnotation {
  @Inject
  public EmptyNamedAnnotation(@Named() Prop<String> myProp) {

  }
}

File: prop-all/src/test/kotlin/io/pleo/prop/objects/EmptyNamedAnnotation.kt
package io.pleo.prop.objects

import com.google.inject.Inject
import io.pleo.prop.core.Prop
import jakarta.inject.Named

@Suppress("UNUSED_PARAMETER")
class EmptyNamedAnnotation
@Inject
constructor(
    @Named myProp: Prop<String>,
)
